tests_new/integration_tests/core/environment/global_conftest.py

```diff
@@ -195,7 +200,25 @@ def persistent_cross_acc_env_1(client5, group5, testdata):
 
 
 @pytest.fixture(scope='session')
-def updated_persistent_cross_acc_env_1(client5, group5, persistent_cross_acc_env_1):
+def updated_persistent_cross_acc_env_1(client5, group5, client6, group6, persistent_cross_acc_env_1):
+    if list_organizations(client6).count == 0:
+        invite_team_to_organization(client5, persistent_cross_acc_env_1.organization.organizationUri, group6)
+    if list_environments(client6).count == 0:
```

Comment:
if org/env have non-zero count because of some other change (i.e a manual change or some other test changing it) `group6` will end up not invited.
I think it would be better if you invite_to_org/invite_to_env and then catch the errors if any (i.e already part of the env/group)

Reply:
I added an explicit check by uri